Project_19/Podfile

```diff
@@ -9,7 +9,7 @@ target 'Project_19' do
 pod 'Firebase/Core'
 pod 'Firebase/Auth'
 pod 'Firebase/Firestore'
-pod 'Firebase/Storage' 
+pod 'Firebase/Storage'
```

Comment:
I recently found that the Pod lines changed and these are outdated. You will find the updated commands [here](https://firebase.google.com/docs/ios/setup).